content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -364,23 +320,45 @@ const getUser = await prisma.user.findUnique({
 })
 ```
 
-### Relation count
+## Relation count
 
 In [3.0.1](https://github.com/prisma/prisma/releases/3.0.1) and later, you can [`include` or `select` a count of relations](aggregation-grouping-summarizing#count-relations) alongside fields - for example, a user's post count.
 
-### Filter a list of relations
+<CodeWithResult>
+<cmd>
 
-When you use `select` to return a subset of data from related records (for example, a user's posts), you can **filter and sort that list of relations**. For example, you might want to query all users with a `prisma.io` email address and select the titles of their published posts.
+```ts
+const relationCount = await prisma.user.findMany({
+  include: {
+    _count: {
+      select: { posts: true },
+    },
+  },
+})
+```
+
+</cmd>
+<cmdResult>
+
+```code no-copy
+{ id: 1, _count: { posts: 3 } },
+{ id: 2, _count: { posts: 2 } },
+{ id: 3, _count: { posts: 2 } },
+{ id: 4, _count: { posts: 0 } },
+{ id: 5, _count: { posts: 0 } }
+```
+
+</cmdResult>
+</CodeWithResult>
+
+## Filter a list of relations
 
-#### Return _all_ `User` records where `email` contains `prisma.io`, and select the `title` of all `Post` records that have not been published yet
+When you use `select` or `include` to return a subset of the related data, you can **filter and sort the list of relations** inside the `select` or `include`.
 
-```js highlight=9-14;normal
+For example, the following query returns all users and a list of published posts associated with each user:
```

Comment:
I think the below query fetches unpublished posts, not published ones. It also only selects `title`

Reply:
Thanks for catching that. I'll make the update. I missed it